doc/parameters/param_list.md

```diff
@@ -5,6 +5,9 @@
 | Parameter | Default | Description |
 | ----------| --------| ------------|
 | default_bt_xml_filename | N/A | path to the default behavior tree XML description |
+| enable_groot_monitoring | False | enable Groot live monitoring of the behavior tree |
```

Comment:
I thought we were going to have this on by default? Any run-time compute reason not to?

Reply:
Groot and ultimately the zmq stuff behind is not adding any mention-able delays (1-1,5ms).
So yes. Having it default `true` should be fine.

I'll change it on all occasions